.gitmodules

```diff
@@ -76,3 +76,9 @@
 [submodule "scripts/packages/Revise"]
 	path = scripts/packages/Revise
 	url = https://github.com/timholy/Revise.jl
+[submodule "scripts/packages/libblastrampoline_jll"]
+	path = scripts/packages/libblastrampoline_jll
+	url = https://github.com/JuliaBinaryWrappers/libblastrampoline_jll.jl.git
+[submodule "scripts/packages/SuiteSparse_jll"]
+	path = scripts/packages/SuiteSparse_jll
+	url = https://github.com/JuliaBinaryWrappers/SuiteSparse_jll.jl.git
```

Comment:
Why would we have to add more submodules?

Reply:
It was an attempt to solve the fact that older versions of Julia could not find them. But I believe the reason for that is that these packages actually do not have a version compatible with older julia versions, so the whole approach was doomed anyway.